container/features/src/main/resources/features-minion.xml

```diff
@@ -8,10 +8,16 @@
     <!-- OpenNMS Features -->
     <repository>mvn:org.opennms.karaf/opennms/${project.version}/xml/features</repository>
 
+    <!-- TODO: Maybe this is already defined somewhere else -->
+    <feature name="minion-core-api" description="OpenNMS :: Minion :: Core :: API" version="${project.version}">
```

Comment:
Yes, this feature should be there. I'd like to do that after this PR is merged because we need to write Minion-only ITs inside the features/minion repo project anyway.